shipyard_blueprints/microsoft-sharepoint/shipyard_microsoft_sharepoint/utils.py

```diff
@@ -0,0 +1,29 @@
+from shipyard_templates import ShipyardLogger, ExitCodeException, CloudStorage
+
+logger = ShipyardLogger.get_logger()
+
+
+def get_credential_group(args):
+    if args.access_token:
```

Comment:
this is most likely going to have to be adjusted to be `if access_token:=os.getenv("OAUTH_ACCESS_TOKEN"): return {"access_token":access_token}` but since the blueprint already had a flag for the access_token figured it would be worth attempting.